SourceCode/include/tracer/RayTracer.h

```diff
@@ -2,28 +2,20 @@
 #include <string>
 #include <vector>
 
-#include "Image.h"
 #include "Ray.h"
-#include "SceneParser.h"
-#include "Triangle.h"
+#include "SceneSettings.h"
 #include "tracer/Camera.h"
 
 class RayTracer {
  private:
   bool rayUpdateRequired;
-  Image image;
-  Camera camera;
-  SceneParser sceneParser;
-  std::vector<Triangle> sceneTriangles;
+  Scene scene;
```

Comment:
If you add an Application module it will be responsible to hold the Scene and pass a pointer to it to the renderer